.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
```

Comment:
Based on the current code style, want to use double quotes `"` by default instead of single `'`. I prefer double for normal string as it's harder to get them mixed up with template string backticks ``` ` ```

Reply:
yeah, sure, seems like I failed to cherry pick that. Probably was commit with huge diff that was fixing linting